### PR TITLE
mrlearning Base Module Update to MRTK 2.1.0

### PR DESCRIPTION
**Pull request by Developer-OI**
Updated Section 2. Initializing your project and first application to MRTK 2.1.0

### CODE REVIEW

File: mixed-reality-docs/mrlearning-base-ch1.md [hunk 1]
```diff
@@ -24,106 +24,107 @@ In this first lesson, you'll learn about some of the capabilities the [Mixed Rea
...
 ### Configure the Mixed Reality Toolkit
 
 1. Add the *Mixed Reality Toolkit* to your current scene by selecting **Mixed Reality Toolkit** > **Add to Scene and Configure..** from the menu bar. If you don't see this menu item after importing the mixed reality toolkit, please restart Unity.
-  ![Lesson1 Chapter3 Step1](images/Lesson1Chapter3Step1.JPG)
+  ![Lesson1 Section4 Step1](images/mrlearning-base-ch1-4-step1.JPG)
 
-> [!NOTE]
-> You may see a pop-up dialog box asking to select a [profile for the Mixed Reality Toolkit](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Profiles/Profiles.html). If so, select **Ok**, and choose the profile named *"DefaultMixedRealityToolkitConfigurationProfile"*.
+    > [!NOTE]
+    > You may see a pop-up dialog box for selecting a [profile for the Mixed Reality Toolkit](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Profiles/Profiles.html). Choose the profile named *DefaultMixedRealityToolkitConfigurationProfile* by double-clicking it.
```
Comment (jessemcculloch) [inline mixed-reality-docs/mrlearning-base-ch1.md]: We should be directing people to use the Default HoloLens 2 profile here

**Comment by mr0ng** [comment]
@grbury looks like I don't have permission to add you as an optional reviewer, but tagging you here per our conversation on the phone. Thanks!

File: mixed-reality-docs/mrlearning-base-ch1.md (hunk 1 above)
Comment (mr0ng) [reply mixed-reality-docs/mrlearning-base-ch1.md]: Sounds good @jessemcculloch

Comment (martinwilter) [reply mixed-reality-docs/mrlearning-base-ch1.md]: The requested change has been completed. Tutorial no prompts people to use the Default HoloLens 2 Configuration Profile, ref commit [494e38f](https://github.com/Developer-OI/mixed-reality/commit/494e38f972eaf02f49f4f1ea2c19a20c7ea70a9b).

**Pull request merged**